packages/expo-file-system/ios/FilePickingUtils.swift

```diff
@@ -77,25 +74,25 @@ internal func createFilePicker(initialUri: URL?, mimeType: String?) -> UIDocumen
       utTypes = [UTType.item]
     }
 
-    let picker = UIDocumentPickerViewController(forOpeningContentTypes: utTypes, asCopy: true)
+    let picker = UIDocumentPickerViewController(forOpeningContentTypes: utTypes, asCopy: options.openAsCopy)
```

Comment:
I tend to believe this isn't as simple as flipping a boolean; using this api means you can access files outside your app sandbox, where `NSFileCoordinator` comes in to play, and overall this can get quite involved.

Maybe it works but.. this needs to be tested on a real device, as emulators don't quite capture the real behavior of this.